data/en/getdirectoryfrompath.json

```diff
@@ -14,5 +14,14 @@
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/function/getdirectoryfrompath"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/getdirectoryfrompath"}
 	},
-	"links": []
+	"links": [],
+	"examples":[
+		{
+			"title":"Basic Example",
+			"description":"",
+			"code":"directoryPath = getDirectoryFromPath(\"C:\\temp\\file.txt\");",
+			"result":"The directoryPath variable will contain the string C:\\temp\\",
+			"runnable":false
```

Comment:
Quick thought, if you change the code example to simply this:

getDirectoryFromPath("c:\temp\file.txt")

then set result to c:\temp\ 

This will actually be runnable even on linux, since it is not actually using the file system, and trycf does not block getDirectoryFromPath.

Reply:
Got it, thanks @pfreitag I get how this works now.